code/game/objects/items/melee/baton.dm

```diff
@@ -212,6 +212,7 @@
 		if(!trait_check)
 			target.Knockdown((isnull(stun_override) ? knockdown_time : stun_override))
 		additional_effects_non_cyborg(target, user)
+	SEND_SIGNAL(target, COMSIG_MOB_BATONED)
```

Comment:
add some more arguments for people that could use it in the future, like the baton used and the attacker

Reply:
Wouldn't that just be simple for someone to add in the future if they were to ever need it?